.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        python -m pip install pre-commit
+        pip install .[pre_commit]
 
     - name: Run pre-commit
       run: pre-commit run --all-files || ( git status --short ; git diff ; exit 1 )
```

Comment:
I am not sure what you mean. Pre-commit is run on all files `pre-commit run --all-files`, while the diff is used only to fail the CI in case files were modified by the pre-commit.

Reply:
Of course, sorry about that.